### PR TITLE
Added verbose function geojson route and 80.2% coverage

### PR DESCRIPTION
**Pull request by SheItoon**

### CODE REVIEW

File: lib/api/controllers/geojson_controller.ex [hunk 1]
```diff
@@ -0,0 +1,25 @@
+defmodule NabooAPI.GeoJsonController do
+  use Phoenix.Controller, namespace: NabooAPI, root: "lib/api"
+  alias NabooAPI.Views.Errors
+
+  require Logger
+
+  def validate_route(conn, params) do
```
Comment (djnnvx) [inline lib/api/controllers/geojson_controller.ex]: image

no swagger?

File: lib/api/router/urls.ex [hunk 2]
```diff
@@ -15,6 +15,7 @@ defmodule NabooAPI.Router.Urls do
   scope "/api" do
     pipe_through(:api)
     post("/login", NabooAPI.SessionController, :sign_in)
+    post("/rfc7946-compliance", NabooAPI.GeoJsonController, :validate_route)
```
Comment (djnnvx) [inline lib/api/router/urls.ex]: please put this on the scope which is piped through `:auth` and `:api`

**Review by djnnvx — CHANGES_REQUESTED** [review]
2 minor fixes and you're good, good job!

File: lib/api/router/urls.ex (hunk 2 above)
Comment (SheItoon) [reply lib/api/router/urls.ex]: it does not require the sender to be authenticated

Comment (djnnvx) [reply lib/api/router/urls.ex]: unauthenticated sender has no use for this though

Comment (SheItoon) [reply lib/api/router/urls.ex]: they can use this route to pre-check their geojson data before actually sending it

Comment (djnnvx) [reply lib/api/router/urls.ex]: don't they need to be authenticated to send the data ?

**Pull request merged**